micro-rdk/src/esp32/entry.rs

```diff
@@ -164,7 +153,16 @@ pub fn serve_web(
     let exec = Esp32Executor::new();
     let cloned_exec = exec.clone();
 
-    let fut = cloned_exec.run(Box::pin(serve_web_inner(
+    cloned_exec
```

Comment:
Neat! A few questions though:
- Is this the general shape of things for periodic tasks? For instance, data sync?
- Why do you need to `clone` the executor above after `new`ing it?

Reply:
we don't really need to that anymore has we can access it from thread_local_storage. I haven't found a definitive answer as too whether executors should be passed around of if they should be sourced from the current thread context
We don't really need to clone it's more to emphasize that it is the same underlying object